cmd/webhook/main.go

```diff
@@ -110,9 +108,7 @@ func NewValidationAdmissionController(ctx context.Context, cmw configmap.Watcher
 		"validation.webhook.serving.knative.dev",
 
 		// The path on which to serve the webhook.
-		// TODO(mattmoor): This can be changed after 0.11 once
-		// we have release reconciliation-based webhooks.
-		"/",
+		"/resource-validation",
```

Comment:
well, why not just `validation`?

Reply:
We also have `/config-validation`, so to disambiguate.